overlay/master.cpp

```diff
@@ -257,21 +252,21 @@ struct Vtep
 
   void reset()
   {
-    freeIP = IntervalSet<IPWrapper>();
+    freeIP = IntervalSet<IP>();
```

Comment:
We still going to do the `.begin` and `.end` methods to simply IPv4 vs IPv6 logic here?